### ISSUE TITLE
inference on squat

### ISSUE THREAD

**Issue by dandingol03** [yuanyuanli85/Stacked_Hourglass_Network_Keras#33]
Hi, thanks for your amazing work. when i inference the pre-trained model on squat image below
![squat02](https://user-images.githubusercontent.com/13013579/101328129-97f5db80-38aa-11eb-85f5-5a3538ceba88.jpg)
the l-hip and r-hip point are missed, because i set the confidence threshold is 0.1,the l-hip and r-hip confidence are below 0.1. Here is the result 
![xxx1](https://user-images.githubusercontent.com/13013579/101328927-a1337800-38ab-11eb-9699-48c6d81fdd70.jpg)

what can i do to make hourglass to recognize the hip part or the bottom-torso when the hip part is being  shielded, thanks for advance

**Comment by yuanyuanli85** [comment]
The hip parts are occluded, which challenges the model's capability.  One way to overcome this is that put multiple cameras so they have chance to see the occluded parts.  Another way is adding more human labels into the training dataset to explicitly tell the model to recognize that part.

**Comment by dandingol03** [comment]
thanks for your wonderful reply.赞！
